nilearn/plotting/img_plotting.py

```diff
@@ -605,6 +618,8 @@ def plot_stat_map(stat_map_img, bg_img=MNI152TEMPLATE, cut_coords=None,
             Specifies whether the colorbar should range from -vmax to vmax
             or from 0 to vmax. Setting to 'auto' will select the latter if
             the whole image is non-negative.
+        vmax: float
+            vmax passed to plt.imshow
```

Comment:
You are aware than "function argument passed on to" gets the quote removed, right? The ratio is way too good to be true.

Not sure how robust these kind of comparisons in general are but I get:

"passed on to" python
167 000 results

"passed to" python
470 000 results